ethcore/src/trace/executive_tracer.rs

```diff
@@ -18,6 +18,7 @@
 
 use ethereum_types::{U256, Address};
 use vm::{Error as VmError, ActionParams};
+use log::{debug, warn};
```

Comment:
Is `debug` used?

Reply:
yes, but `stable` branch uses 2015 edition with `#[macro_use] extern crate` style imports